src/Catty/Setup/CatrobatSetup+Bricks.swift

```diff
@@ -178,6 +180,12 @@
                                             color: UIColor.arduinoBrick,
                                             strokeColor: UIColor.arduinoBrickStroke))
         }
+        if isEmbroideryEnabled() {
```

Comment:
If the embroidery extension is enabled, please add as first category (above "Event").